app/policies/application_policy.rb

```diff
@@ -3,9 +3,10 @@ class ApplicationPolicy
 
   # In your controller, Pundit will call the current_user method
   #  to retrieve what to send into the argument
-  def initialize(user, resource)
-    @user = user
-    @resource = resource
+  def initialize(user, record)
+    raise Pundit::NotAuthorizedError, "must be logged in" unless user
+    @user   = user
+    @record = record
```

Comment:
This causes some variable name problems that I list out in the overall comment thing. This note is a reminder to look at those

Reply:
OH yeah this was an accident I'll change it back